stdlib/LinearAlgebra/test/factorization.jl

```diff
@@ -0,0 +1,43 @@
+@testset "equality for factorizations - $f" for f in Any[
+    bunchkaufman,
+    cholesky,
+    x -> cholesky(x, Val(true)),
+    eigen,
+    hessenberg,
+    lq,
+    lu,
+    qr,
+    x -> qr(x, ColumnNorm()),
```

Comment:
This needs to be manually changed to `qr(x, Val(true))` when backporting this to 1.6. @KristofferC what is the best way to ensure this? Should I maybe remove the `backport-1.6` label here and open a new PR with that change against the `release-1.6` branch?